packages/api/cms-api/src/generator/crud-generator.decorator.ts

```diff
@@ -1,24 +1,27 @@
 export interface CrudGeneratorOptions {
     targetDirectory: string;
+    requiredPermission?: [string];
```

Comment:
`[string] | string` would be appropriate here, as in most cases you'll use a single string and the option is named in singular form